extensions/Worker.Extensions.ServiceBus/src/ServiceBusReceivedMessageConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.Amqp;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+[SupportsDeferredBinding]
+[SupportedConverterType(typeof(ServiceBusReceivedMessage))]
+[SupportedConverterType(typeof(ServiceBusReceivedMessage[]))]
```

Comment:
Just to double check, do we not need IEnumerable<> support now? I thought we don't need IList<> but still need IEnumerable<>.

Reply:
IEnumerable does not work for in-proc, so I removed it here.